encodings/runend/src/array.rs

```diff
@@ -235,6 +235,49 @@ impl StatisticsVTable<RunEndArray> for RunEndEncoding {
                     .unwrap_or(false)
                     && array.logical_validity().all_valid(),
             )),
+            Stat::TrueCount => match array.dtype() {
+                DType::Bool(_) => {
```

Comment:
you still need to subtract nulls?

Reply:
Ah, good catch, fixed.

Reply:
I'm gonna extract this to a separate PR.

Reply:
https://github.com/spiraldb/vortex/pull/2007